src/renderer/state.ts

```diff
@@ -208,6 +208,34 @@ export class AppState {
     versions: this.baseVersions,
   });
 
+  // Lock on the active Electron version that prevents other windows from removing it
+  private versionLock: Lock | null = null;
+
+  // Used to release the lock when the current window switches Electron versions
+  private versionLockController = new AbortController();
+
+  private static versionLockNamePrefix = 'version:';
+
+  public getVersionLockName(ver: string) {
+    return [AppState.versionLockNamePrefix, ver].join('');
```

Comment:
```suggestion
    return `${AppState.versionLockNamePrefix}${ver}`;
```

Seems more intuitive, IMO.